tools/distrib/generate_grpc_shadow_boringssl.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright 2018 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+cd $(dirname $0)
+
+filename_original="../../src/core/tsi/grpc_shadow_boringssl.h"
+boringssl_podspec_original="../../src/objective-c/BoringSSL-GRPC.podspec"
+if [ "$TEST" == "" ]; then
+  filename=$filename_original
+  boringssl_podspec=$boringssl_podspec_original
+else
+  filename="${filename_original}.tmp"
+  boringssl_podspec="${boringssl_podspec_original}.tmp"
+fi
+symbol_list="../../src/objective-c/grpc_shadow_boringssl_symbol_list"
+
+symbols=($(cat $symbol_list | tail -n +2))
+
+> $filename
+echo '/*' >> $filename
```

Comment:
nit: do we really need to generate the license agreement? it adds complexity to the file and obscures the real purpose.

Reply:
I'm afraid the license sanity checker would complain if we do not add that.